src/envo/envo-edit.owl

```diff
@@ -27848,13 +27869,157 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000000> <http://purl.obolibrar
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_03000002> (firn)
 
-AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000002> "Ice which is an intermediate between snow and glacial ice and which is formed by the compaction and recrystalisation of snow melted from névé.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000002> "Granular snow which contains a proportion of fused granules formed through melting and re-thawing while being compacted by the weight of overlying material and which contains interconnected gaseous pores.")
 AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000117> <http://purl.obolibrary.org/obo/ENVO_03000002> "http://orcid.org/0000-0002-3410-4655")
 AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000119> <http://purl.obolibrary.org/obo/ENVO_03000002> "https://en.wikipedia.org/wiki/N%C3%A9v%C3%A9")
-AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03000002> "Firn derives from Névé that survives a full season of ablation.")
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03000002> "Firn derives from névé that survives a full season of ablation, which undergone some metamorphic transition.")
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03000002> "firn"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03000002> <http://purl.obolibrary.org/obo/ENVO_01000293>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_03000003> (nivation)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03000003> "A material transformation process occurring due to a mass wasting process, or by cycles of freezing and thawing in which fresh snow is compacted into firn or névé.")
```

Comment:
The "fresh" qualifier is a bit problematic. We're trying to communicate that little to no metamorphosis has occurred since the snow was deposited on the surface. Perhaps work that into the def instead of "fresh"